admin/email/email.go

```diff
@@ -35,3 +45,13 @@ func (c *Client) SendProjectInvite(toEmail, toName, projectName, roleName string
 	)
 	return err
 }
+
+func (c *Client) SendProjectAdditionNotification(toEmail, toName, projectName, roleName string) error {
+	err := c.sender.Send(
+		toEmail,
+		toName,
+		"Addition to a project",
+		fmt.Sprintf("You've been added to project <b>%s</b> as <b>%s</b>. Follow this <a href=\"%s\">link</a> to visit it.", projectName, roleName, c.frontendURL),
```

Comment:
1. Put "the" before "project"
2. Format project name as `orgName/projectName`

Reply:
orgName not present, will need to be fetched using orgId